examples/scalajs-react/src/main/scala/todomvc/CTodoList.scala

```diff
@@ -31,34 +33,45 @@ object CTodoList {
    *
    * It extends OnUnmount so unsubscription of events can be made automatically.
    */
-  case class Backend($: BackendScope[Props, State]) extends OnUnmount {
+  class Backend($: BackendScope[Props, State]) extends OnUnmount {
 
-    def handleNewTodoKeyDown(event: ReactKeyboardEventI): Option[IO[Unit]] =
-      Some((event.nativeEvent.keyCode, UnfinishedTitle(event.target.value).validated)) collect {
-        case (KeyCode.Enter, Some(title)) =>
-          IO(event.target.value = "") |+| $.props.model.addTodo(title)
-      }
-    
-    def updateTitle(id: TodoId)(title: Title): IO[Unit] =
-      editingDone(cb = $.props.model.update(id, title))
+    /**
+     *  A backend lives for the entire life of a component. During that time,
+     *   it might receive new Props,
+     *   so we use this mechanism to keep state that is derived from Props, so
+     *   we only update it again if Props changed in a meaningful way (as determined
+     *   by the implicit `Reusability` defined above )
+     */
+    case class Callbacks(P: Props){
+      val handleNewTodoKeyDown: ReactKeyboardEventI => Option[Callback] =
+        e => Some((e.nativeEvent.keyCode, UnfinishedTitle(e.target.value).validated)) collect {
+          case (KeyCode.Enter, Some(title)) =>
+            Callback(e.target.value = "") >> P.model.addTodo(title)
+        }
+
+      val updateTitle: TodoId => Title => Callback =
+        id => title => editingDone(cb = P.model.update(id, title))
+
+      val toggleAll: ReactEventI => Callback =
+        e => P.model.toggleAll(e.target.checked)
+    }
 
-    def startEditing(id: TodoId): IO[Unit] =
-      $.modStateIO(_.copy(editing = Some(id)))
+    val cbs = Px.cbA($.props).map(Callbacks)
+
+    val startEditing: TodoId ⇒ Callback =
+      id ⇒ $.modState(_.copy(editing = Some(id)))
```

Comment:
i fully approve of that as a word. Yeah, this is the moment when i tell my IDE to stop autounicodifying those arrows (how about that for a word?)